src/Plotting/paraview.jl

```diff
@@ -0,0 +1,49 @@
+function Base.convert(::Type{<:VTKPolyhedron}, X::LazySet; shift=0)
+
+    dim(X) == 3 || throw(ArgumentError("the dimension of the set should be three, got $(dim(X))"))
+
+    poly = polyhedron(convert(HPolytope, X))
```

Comment:
`HPolyhedron` would be more general here.

Reply:
true; i changes this to `HPolyhedron`.

also had to add convert methods from a general lazyset that were missing.